var/spack/repos/builtin/packages/liblouis/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Liblouis(AutotoolsPackage):
+    """Liblouis is an open-source braille translator and back-translator
+    named in honor of Louis Braille."""
+
+    homepage = "http://liblouis.org/"
+    url      = "https://github.com/liblouis/liblouis/releases/download/v3.15.0/liblouis-3.15.0.tar.gz"
+
+    version('3.15.0', sha256='3a381b132b140747e5fcd47354da6cf43959da2167f8bc598430bbac51224467')
```

Comment:
The latest version seems to be v3.17.0 Can you please add that too?

Reply:
Added this version in: https://github.com/spack/spack/pull/22141/commits/504e5779c088c1419b0811c098f1328df70d74f7